### PR TITLE
Add watchfacespreview

### PR DESCRIPTION
**Pull request by eLtMosen**
Previews for all current watchfaces but analog-words and 001-words-worte-palabras-mots.qml. Since those two should be displayed localised.
- current folder size is 5.5MB
- compression of all png at 70% to preserve visually quality
- image resolutions are Dims.x(40) of the currently supported resolutions

### CODE REVIEW

**Comment by FlorentRevest** [comment]
As discussed in https://github.com/AsteroidOS/unofficial-watchfaces/pull/58 I’m a bit uneasy about the 40% width but if you say it’s fine, then it is approved for merging ;)

**Review by FlorentRevest — APPROVED** [review]
Where will this .watchfacespreview directory end up ? If it ends up in a directory that is itself named watchfaces, I’d recommend we rename this to .preview instead. The context of what previews these are should be clear enough already.

**Comment by eLtMosen** [comment]
> As discussed in #58 I’m a bit uneasy about the 40% width but if you say it’s fine, then it is approved for merging ;)

The reasoning is solid for this version with pre supplied previews imo
- 40% screen size is the height of each Grid item in the watchfacePage. That is to ensure consitency with the wallpaperPage where wallpapers are also not presented in full height to shorten the height of the full grid-list. The wallpapers however are placed in the grid items using PreserveAspect. Which results in 50% Dims images that are "cut off" by 20% height to fit the Dims.x(40) Gid height.
- Since watchfacepreviews are placed in that same grid dimensions (for consistency over the two pages) using Image.PreserveAspectFit, also the width will always be 40% Dims when keeping aspect ratio.
- That has never changed as of yet and during redesign it did not occur to me to change it now. Since 40% grid item height is a perfect trade off between length of the complete grid, number of row visible on screen at a time and still legible image size at lowest resolution. 

In summary, the 40% depend on both, wallpaper- and watchfacePage. And only a complete redesign of both layouts would make an adaption of the resulting preview image resolutions necessary.

**Comment by eLtMosen** [comment]
> Where will this .watchfacespreview directory end up ? If it ends up in a directory that is itself named watchfaces, I’d recommend we rename this to .preview instead. The context of what previews these are should be clear enough already.

My hope was we could add a .bb file to only copy that `.watchfacespreview` folder to `/usr/share/asteroid-launcher/watchfacespreview`for the builds?
That would keep it simple for contributors to be able to PR a watchface into one repo only, not needing a second PR to place  the previews to a different repo?
Putting only the `preview` trunc into the `watchfaces` folder results in the watchfacePage displaying on empty grid item per preview image.
And sorry for not asking about the folder structure upfront. It was just late last night
The current state is what resulted by mitigating above two issues to place it somewhere else.
But at least i learned and waited with doing the watchfacePage PR until the folder structure decision is made.
Happy to follow your advise.

**Comment by eLtMosen** [comment]
> Where will this .watchfacespreview directory end up ? If it ends up in a directory that is itself named watchfaces, I’d recommend we rename this to .preview instead. The context of what previews these are should be clear enough already.

Converting to draft since i made a major thinking error.
The assets for the WF are all located in the individual WF folders. I am moving them there now individually.
This will make the deploy script just copy them individually per watchface.
Instead of us caring how they end up in asteroid-launcher.
